packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -85,22 +79,20 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
       <ModalRow>
         <ModalLabel labelKey="label.item" />
         <Grid item flex={1}>
-          <ItemSearchInput value={selectedItem} onChange={onChangeItem} />
+          <ItemSearchInput
+            currentItemId={summaryItem?.itemId}
+            onChange={onChangeItem}
+          />
         </Grid>
       </ModalRow>
       <ModalRow>
         <ModalLabel labelKey="label.available" />
-        <NumericTextInput
-          defaultValue={invoiceLine?.itemCode}
-          inputProps={register('availableQuantity')}
-          disabled
-          sx={{ width: 85 }}
-        />
+        <NumericTextInput value={0} disabled sx={{ width: 85 }} />
```

Comment:
there was a couple of these I was a little confused about. Seems this is just a disabled text input so I was unsure why it needed to be registered. Seems the `setValue` isn't really needed as we pass a different item the component rerenders.

Reply:
Ah, also it's being set to just `0` which is likely not ideal!

This will change to be the sum of the pack size selected?

Reply:
it was `item.availableQuantity` and I think the value should just show that, without doing anything fancy with pack size - the API can do the calculating.
I suppose the availableQuantity must be unit quantity, right?

Reply:
Heh. Just looking at this now.

- Available quantity is something we added and isn't in the 'real' api.
- We also might need to calculate this on the client, anyway. 
    - If the server sends an available quantity of '100', then we go and delete the line for the item, then go and add that item *again*, without having saved the delete, should we add the deleted line packs into the available quantity?
- I thought we had said to change the available quantity based on the pack size selected? i.e. if the total unit quantity is 1000, spread over 5 packs of 100 and 10 packs of 50, then when 50 is selected the available is 10 (and is the maximum which can be entered into the pack quantity input)

What I was heading toward:

```
So, say there are stock lines for a pack size of 2 (A), a pack size of 10 (B) and two for a pack size of 5 (C,D). 

Then, if the user selects a pack size of 5, the modal is essentially pretending the other stock lines don’t exist? So, the available quantity is equal to [C]numberOPacks * 5 + [D]numberOPacks * 5 (and maybe grey out the other stock lines?)

Then if the user changes the pack size, to 10, then everything resets, and only [B] matters, with the available quantity being [B]numberOfPacks*10, and only [B] having stock allocatable
```

Reply:
oh, yeah. I see.
Side thought - we should label that 'available number of packs' rather than 'available quantity'

you're right about deleting existing allocation, and how this affects available quantity. Makes sense to sum the available on the client for that alone - also helps in implementing pack size selection by the user.